Client/Cliqz/Foundation/Anti-Phishing/AntiPhishingDetector.swift

```diff
@@ -21,7 +21,13 @@ class AntiPhishingDetector: NSObject {
             completion(false)
             return
         }
-		let queue = DispatchQueue.global(priority: DispatchQueue.GlobalQueuePriority.background)
+        
+        if isDetectedPhishingURL(url) {
```

Comment:
what is wrong with indentation in this file?

Reply:
I don't know exactly, but some times github does not recognize the indentation properly, anyway the problem was in the removed line and the added code is correctly indented